aviary/interface/methods_for_level2.py

```diff
@@ -604,11 +604,9 @@ def _get_2dof_phase(self, phase_name):
         if 'cruise' not in phase_name:
             phase.add_control(
                 Dynamic.Mission.THROTTLE, targets=Dynamic.Mission.THROTTLE, units='unitless',
-                opt=False, lower=0.0, upper=1.0
```

Comment:
why are we removing these limits?

Reply:
not optimizer-controlled so limits don't mean nothing